docs/proposal/20240308-vpc-infra-creation.md

```diff
@@ -0,0 +1,393 @@
+# Dynamically create infrastructure required for VPC cluster
+
+## Motivation
+Currently, inorder to create a VPC cluster using cluster api we need to create
```

Comment:
Currently capi does create various resources automatically, like vpc, subnet, loadbalancer so on, May be we can rephrase the sentence or mention the resources which we need to create manually.

Reply:
+1 the present CAPIBM flow for VPC creates everything on its own.

Note - can you please elaborate here which prerequisites you are referring too?

Reply:
Sure, I can attempt to make things a bit clearer.

Reply:
Updated, hopefully that makes things a bit easier to follow and explains what we are trying to do, expanding the current VPC support. options.